rpc/documentation/api.md

```diff
@@ -1559,6 +1560,42 @@ signature algorithms other than secp256k1.
 
 ___
 
+#### `ValidateAddress`
+
+The `ValidateAddress` method verifies if an address is valid.
+
+**Request:** `ValidateAddressRequest`
+
+- `string address`: The address to be validated.
+
+**Response:** `ValidateAddressResponse`
+
+- `bool is_valid`: True if valid, false if not.
+
+- `string address`: The validated address.
+
+- `bool is_mine`: True if the address is an address of querying wallet, false if not.
+
+- `bool is_watch_only`: True if the querying wallet is watch-only, false if not.
```

Comment:
if this info is needed it should be moved out to a different rpc.  Imagine i wanted to check this, now i have to call ValidateAddress with some address from the wallet, but the address isn't actually needed to provide this info.